lib/octokit/client/authorizations.rb

```diff
@@ -152,12 +152,13 @@ def scopes(token = @access_token, options = {})
       #  client = Octokit::Client.new(:client_id => 'abcdefg12345', :client_secret => 'secret')
       #  client.check_application_authorization('deadbeef1234567890deadbeef987654321')
       def check_application_authorization(token, options = {})
-        opts = options.dup
+        opts = ensure_api_media_type(:applications_api, options.dup)
+        opts[:access_token] = token
         key    = opts.delete(:client_id)     || client_id
         secret = opts.delete(:client_secret) || client_secret
 
         as_app(key, secret) do |app_client|
-          app_client.get "applications/#{client_id}/tokens/#{token}", opts
+          app_client.post "applications/#{client_id}/tokens", opts
```

Comment:
Hey @tarebyte,
The paths you are using here and returing 404. According to https://developer.github.com/changes/2020-02-14-deprecating-oauth-app-endpoint/#endpoints-affected , it should be singular `token`, and not `tokens`

Reply:
Damn, thanks for the heads up. I'm working on a PR to resolve this.

Reply:
> I've got the tests passing but I've yet to actually try this against the live API 😄

🤦‍♂️

Reply:
https://github.com/octokit/octokit.rb/releases/tag/v4.18.0

Reply:
Thanks for the quick fix. Seems to be working correctly now :)